net.agalwood.Motrix-git.json

```diff
@@ -0,0 +1,5 @@
+{
+    "type": "git",
+    "url": "https://github.com/agalwood/Motrix.git",
+    "ref": "v1.6.10"
+}
```

Comment:
this file isn't referenced anywhere in the manifest.

Reply:
Yes. It's used for auto update the `generated-sources.json`: https://github.com/flathub/flathub/pull/2334/files#diff-3db03b056e588432b22814fac93c11d44c8a4d8d958713cd2bf70b1d1d0ab48dR107